src/dataSources/cloudFirestore/session.js

```diff
@@ -23,7 +24,37 @@ const session = dbInstance => {
     return results;
   }
 
-  return { findAllApprovedByEventId };
+  async function findApprovedBySlug(eventId, slug) {
+    dlog('find in event %s by slug %s', eventId, slug);
+    const docSnap = await sessionsCollections
+      .where('eventId', '==', eventId)
+      .where('status', 'in', ['ACCEPTED', 'SCHEDULED', 'CANCELLED'])
+      .where('slug', '==', slug)
+      .get();
+
+    let result = null;
+
+    if (docSnap.size === 1) {
+      result = docSnap.docs[0].data();
+      result.id = docSnap.docs[0].id;
+    } else if (docSnap.size > 0) {
+      dlog(`Multple sessions return for event ${eventId}, with slug ${slug}`);
+      Sentry.withScope(scope => {
+        scope.setLevel('error');
+        scope.setFingerprint('duplicate_slug');
+        scope.setContent('duplicate_slug', {
+          eventId,
+          slug,
+          duplicate_count: docSnap.size,
+        });
+        Sentry.captureException(new Error('duplicate slugs in event'));
```

Comment:
do we want exception or event?

Reply:
@csell5 That is a good question. I was unable to find the difference in there documentation and went with Exception. I will dig deeper.

Reply:
I think they just show up here but without the "error" 
![image](https://user-images.githubusercontent.com/772569/84198526-4be7ec80-aa69-11ea-9588-c4e876cddb2c.png)

Reply:
Updated using captureMessage(). captureEvent() has very little documentation around it and I am not sure on how it handles the set scope